src/gpu/intel/ocl/reusable_softmax.cpp

```diff
@@ -180,6 +180,20 @@ status_t reusable_softmax_fwd_t::pd_t::init_dispatch_workgroup_per_reduction(
     conf.gws_params = dispatch.get_compile_params();
     rt_conf.gws_params = dispatch.get_runtime_params();
 
+    auto dispatch_lws = dispatch.get_runtime_params().nd_range.local_range();
+    auto dispatch_gws = dispatch.get_runtime_params().nd_range.global_range();
+
+    auto *device_info = compute_engine->device_info();
+    const size_t multiple_of_sg_lws
+            = utils::rnd_up(dispatch_lws[0], device_info->max_subgroup_size());
```

Comment:
A better workaround would be to just set the subgroup size in the dispatch config so that no fixup is required. It seems like this is also a bug in the OpenCL runtime, could you provide a reproducer to them as well?